tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/cache/CacheTest.java

```diff
@@ -75,26 +75,36 @@ public void testPut() {
         args[0] = new BString(key);
         args[1] = new BString(value);
         BValue[] returns = BRunUtil.invoke(compileResult, "testPut", args);
-        Assert.assertTrue(returns.length == 1);
+        Assert.assertEquals(returns.length, 1);
         Assert.assertTrue(returns[0] instanceof BInteger);
         Assert.assertEquals(((BInteger) returns[0]).intValue(), 1);
     }
 
     @Test
-    public void testGet() {
+    public void testGet1() {
```

Comment:
Shall we rename these test methods to more meaningfull ones? like testGettingExistingValue